plugin/src/tryGetObjects.lua

```diff
@@ -0,0 +1,88 @@
+local Fmt = require(script.Parent.Parent.Fmt)
+local Log = require(script.Parent.Parent.Log)
+local PatchSet = require(script.Parent.PatchSet)
+local Promise = require(script.Parent.Parent.Promise)
+
+local function tryGetObjects(instanceMap, apiContext, patch)
+	assert(PatchSet.validate(patch))
+
+	local unappliedPatch = PatchSet.newEmpty()
+
+	-- GetObjects won't help with anything that failed to remove
+	unappliedPatch.removed = patch.removed
+	-- TODO: Implement this? Do we have to?
```

Comment:
Yes! This is important for if a file is newly added on the filesystem

Reply:
Figured, just couldn't figure it out because I was using existing projects, and wasn't creating new files.

Reply:
I copied and pasted a file and...it worked perfectly? It was still under `updated`, not `added`.

Reply:
Nevermind, added seems to be used when modifying .project.json.